SPECS-EXTENDED/libpq/libpq.spec

```diff
@@ -0,0 +1,178 @@
+%global majorversion 12
+
+Summary:        PostgreSQL client library
+Name:           libpq
```

Comment:
It looks like we already have the `libpq` package provided through `postgresql-libs`. Similarly, that core package already builds `postgresql-devel`, so we this will just introduce conflicts/duplicates. What is the purpose of adding this package? If we're missing something in the core's `postgresql`, we might want to extend that existing spec.